tsconfig.json

```diff
@@ -4,13 +4,16 @@
     "emitDecoratorMetadata": true,
     "experimentalDecorators": true,
     "esModuleInterop": true,
-    "lib": ["ESNext", "dom"],
+    "lib": [
+      "ESNext",
+      "dom"
+    ],
     "module": "ESNext",
     "moduleResolution": "node",
     "resolveJsonModule": true,
     "sourceMap": true,
     "target": "es2017",
-    "jsx": "react",
+    "jsx": "react-jsx",
```

Comment:
Did you have red squiggly lines everywhere in the code? I had them because this prop is incorrect and then changing to `react` fixes it. But everytime `yarn start` is run it just swaps it back.

(Not your problem just venting haha)